apps/app/src/routes.ts

```diff
@@ -18,6 +18,9 @@ export default [
   route("/apps/articles", "pages/apps/articles/index.tsx"),
   route("/apps/articles/new", "pages/apps/articles/new.tsx"),
   route("/apps/articles/:id", "pages/apps/articles/[id].tsx"),
+  route("/apps/video-notes", "pages/apps/video-notes/index.tsx"),
+  route("/apps/video-notes/new", "pages/apps/video-notes/new.tsx"),
+  route("/apps/video-notes/:id", "pages/apps/video-notes/[id].tsx"),
```

Comment:
i did ask for this to enhance the existing notes app so will have to re work to do that